test/Dialect/FIRRTL/annotations.mlir

```diff
@@ -1342,3 +1342,62 @@ firrtl.circuit "Top"  attributes {rawAnnotations = [{
     %dut_clock, %dut_reset, %dut_in, %dut_out = firrtl.instance dut interesting_name  @DUT(in clock: !firrtl.clock, in reset: !firrtl.reset, in in: !firrtl.bundle<uint: uint<1>, vec: vector<uint<1>, 2>>, out out: !firrtl.bundle<uint: uint<1>, vec: vector<uint<1>, 2>>)
   }
 }
+
+// -----
+
+
+firrtl.circuit "Top"  attributes {rawAnnotations = [
+  {class = "sifive.enterprise.grandcentral.DataTapsAnnotation",
+  keys = [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
+  source = "~Top|Top/foo:Foo/b:Bar>inv", wireName = "~Top|Top>tap"}]}]} {
```

Comment:
Mega-nit: these `rawAnnotations` could be formatted better as opposed to just line-broken. This is stupid and tedious, but makes the tests easier to read. 😅

Reply:
Reformatted them